### ISSUE TITLE
ToulligQC 2.2 not including graphs in html output file

### ISSUE THREAD

**Issue by smaegol** [GenomiqueENS/toulligQC#9]
Hi,

after installation of the latest version of ToulligQC (from PyPi) images are not included in the HTML report. 

The command:

```
$ toulligqc -a guppy_out_5.0.11/sequencing_summary.txt -o test.html 
test.html
ToulligQC version 2.2
* Initialize extractors
* Start Toulligqc info extractor
* End of Toulligqc info extractor (done in 0m0.00s)
* Start Basecaller sequencing summary extractor
  - Load sequencing summary file (54.05 MB used) in 0m3.21s
  - Extract info from sequencing summary file in 0m10.38s
  - Creation of image "Read count histogram" in 0m0.28s
  - Creation of image "Distribution of read lengths" in 0m4.99s
  - Creation of image "Yield plot through time" in 0m2.62s
  - Creation of image "PHRED score distribution" in 0m4.65s
  - Creation of image "PHRED score density distribution" in 0m1.03s
  - Creation of image "Channel occupancy of the flowcell" in 0m1.11s
  - Creation of image "Correlation between read length and PHRED score" in 0m1.24s
  - Creation of image "Read length over time" in 0m3.15s
  - Creation of image "PHRED score over time" in 0m3.60s
  - Creation of image "Translocation speed" in 0m3.69s
* End of Basecaller sequencing summary extractor (done in 0m39.96s)
* Write HTML report
* Write statistics files
* End of the QC extractor (done in 0m39.98s)
```
produces an HTML file with only Run statistics and Device and software parts.

When run with the --images-directory option, HTML files with graphs are correctly produced in the specified directory but are not shown in the HTML report, so there is no problem with graphs production. 

When running the previous version (2.1.1) on the same input everything is correct. 

Do you have any solution?

System: Ubuntu 20.04, kernel 5.4.0-47
Python: 3.8.5
plotly: 5.5.0
matplotlib: 3.4.3
numpy: 1.19.2

**Comment by jourdren** [comment]
Dear  smaegol, 
Thanks for reporting the issue. I've just fixed the issue (generated images were not added to the image list).
A fixed version will be on line in few hours.
Best,
Laurent.

**Comment by jourdren** [comment]
Issue has been fixes in version 2.2.1.
